lms/templates/dashboard/course.underscore

```diff
@@ -0,0 +1,688 @@
+<div>
```

Comment:
@clintonb I already commented to Afeef around moving a lot of the logic into the view to reduce repetition in this file, but it appears that he didn't do that. @waheedahmed can you please take a look at the comment in this file and move some of the logic into the view.

To answer your actual question @clintonb, the choices are to either break this down into several sub-views, or have the View use several templates to create an object that contains HTML values that are constructed in this template. I have not seen a clean way to do nested templates other than this, but that's not to say that it isn't possible.